bot-twitter/package.json

```diff
@@ -13,6 +13,7 @@
     "start": "nuxt start",
     "generate": "nuxt generate",
     "lint": "eslint --ext \".js,.ts\" --ignore-path .gitignore . --max-warnings=0",
+    "debug": "node --inspect node_modules/.bin/nuxt",
```

Comment:
What does this debug do and what can we use it for in the future? (Just interested)

Reply:
`node --inspect` starts a websockets endpoint, to which you can connect your chrome console and have access to the running program. More info here: https://nodejs.org/en/docs/guides/debugging-getting-started/